lang/java/reef-checkpoint/src/main/java/org/apache/reef/io/checkpoint/CheckpointNamingService.java

```diff
@@ -24,10 +24,10 @@
 public interface CheckpointNamingService {
 
   /**
-   * Generate a new checkpoint Name.
+   * Generates a file name for a new checkpoint.
    *
    * @return the checkpoint name
    */
-  String getNewName();
+  String getNewCheckpointName();
```

Comment:
I will return the previous name here since this renaming also breaks the API which is public.